content/docs/start/visualization.md

```diff
@@ -0,0 +1,113 @@
+# Visualization with Plots
+
+In this section, we'll add visualization to the [`example-dvc-experiments`][ede]
+project. If you would like to try these yourself, please refer to the project
+[README] about how to install.
+
+[ede]: https://github.com/iterative/example-dvc-experiments
+[readme]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/README.md
+
+### Creating plots from text with DVC
+
+A useful plot to show the classification performance is the confusion matrix. In
```

Comment:
Link "confusion matrix" to somewhere? E.g. https://towardsdatascience.com/understanding-confusion-matrix-a9ad42dcfd62